plutus-scb/src/Plutus/SCB/Core/ContractInstance.hs

```diff
@@ -67,24 +73,76 @@ import           Plutus.SCB.Events.Contract                      (ContractEvent
 import qualified Plutus.SCB.Events.Contract                      as Events.Contract
 import qualified Plutus.SCB.Query                                as Query
 import           Plutus.SCB.Types                                (SCBError (..), Source (ContractEventSource, NodeEventSource, UserEventSource, WalletEventSource))
-import           Plutus.SCB.Utils                                (render, tshow)
+import           Plutus.SCB.Utils                                (tshow)
 
 import qualified Plutus.SCB.Core.Projections                     as Projections
 
+data ContractInstanceMsg =
+    ProcessFirstInboxMessage ContractInstanceId (Response ContractResponse)
+    | SendingContractStateMessages (Doc Void) IterationID [Request ContractSCBRequest]
```

Comment:
Again... seems like we may just end up with that `t` parameter around at some point anyway!